nuxeo/templates/ingress.yaml

```diff
@@ -38,7 +38,14 @@ spec:
     {{- with .Values.ingress.hostname }}
     host: {{ toYaml . }}
     {{- end }}
-  {{- with .Values.ingress.tls }}
-  tls: {{- toYaml . | nindent 4 }}
+  {{- if .Values.ingress.tls }}
+  tls: {{- toYaml .Values.ingress.tls | nindent 2 }}
+  {{- else if .Values.ingress.singleTls.secretName }}
+  tls:
+  - secretName: {{ .Values.ingress.singleTls.secretName }}
+    {{- with .Values.ingress.hostname }}
+    hosts:
+    - {{ toYaml . }}
+    {{- end }}
```

Comment:
This is because the GCP marketplace [app](https://github.com/nuxeo/nuxeo-gcp-marketplace) relies on user-provided strings (UI form defined by schema.yaml) that are mapped to the Helm chart's values, so we cannot handle a list here.
Yet, we've found a more generic way.